lib/jeckle/mapping.rb

```diff
@@ -0,0 +1,30 @@
+module Jeckle
+  module CustomAttributeMapping
+    def self.included(base)
+      base.send :extend, Jeckle::CustomAttributeMapping::Methods
+    end
+
+    module Methods
+      def mapping(&block)
+        @mapping ||= AttributeMapping.new(self)
+
+        @mapping.instance_eval(&block) if block.present?
```

Comment:
We don't need, but if we don't check for the block presence, a miss mapping without a block will cause an exception. It is just to make sure that developers don't mess up things by typos.

Better use block_given!